regexes/oss.yml

```diff
@@ -1094,17 +875,9 @@
   name: 'HP-UX'
   version: '$1'
 
-- regex: 'FreeBSD(?:[/ ]?(\d+[\.\d]+))?'
-  name: 'FreeBSD'
-  version: '$1'
-
-- regex: 'NetBSD(?:[/ ]?(\d+[\.\d]+))?'
-  name: 'NetBSD'
-  version: '$1'
-
-- regex: 'OpenBSD(?:[/ ]?(\d+[\.\d]+))?'
-  name: 'OpenBSD'
-  version: '$1'
+- regex: '(Free|Net|Open)BSD(?:[/ ]?(\d+[\.\d]+))?'
+  name: '$1BSD'
+  version: '$2'
```

Comment:
Not sure if we should combine them. This might cause that the OS is returned in different writing cases, depending how it was written in the user agent.

Reply:
returned the changes back